serverless.ts

```diff
@@ -135,6 +140,19 @@ const serverlessConfiguration: AWS = {
               "dynamodb:DeleteItem",
             ],
             Resource: isLocal ? "*" : "arn:aws:dynamodb:${self:provider.region}:*:table/Questionnaires"
+          },
+          {
+            Effect: "Allow",
+            Action: [
+              "dynamodb:DescribeTable",
+              "dynamodb:Query",
+              "dynamodb:Scan",
+              "dynamodb:GetItem",
+              "dynamodb:PutItem",
+              "dynamodb:UpdateItem",
+              "dynamodb:DeleteItem",
+            ],
+            Resource: isLocal ? "*" : "arn:aws:dynamodb:${self:provider.region}:*:table/VacationRequests"
```

Comment:
I dont see how exactly much different from the way we did it, besides the recources. Could you elaborate on that?

Reply:
Here we seem to be specifying the actions array for each resource/ table. The example I linked is setting the same actions for all the resources but without having to repeat the 7 lines of "actions" like we are here. At least I think this is what it is doing, you can check the serverless docs to ensure this what it is doing.

 As we can expect the number of tables to increase it would be nice if we could just add a single line for the resource rather than repeat the actions array for each one. We could also go through this together later